### DISCUSSION TITLE
I just want to ask what signal the terminal sends on close by clicking on the close button (x) (without ctrl+c)? Does it send anything at all? How to make it send it?

### DISCUSSION THREAD

**Discussion by moneyDev1111** [git-for-windows/git#4869] [Q&A] [answered]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?
 **
 2.44.0.windows.1 64 bit
 **
```
$ git --version --build-options
**
cpu: x86_64
built from commit: ad0bbfffa543db6979717be96df630d3e5741331
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon
**
 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?
 **
windows 11 x64
**
```
$ cmd.exe /c ver
**
Microsoft Windows [Version 10.0.22631.3296]
(c) Microsoft Corporation. All rights reserved.
**
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
> type "$env:USERPROFILE\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

** Editor Option: VisualStudioCode
Custom Editor Path: 
Default Branch Option:  
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled
 **
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

** no **

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

** none **

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
** none **
```
 - What did you expect to occur after running these commands?

** **

 - What actually happened instead?

** **

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

**  **

**Comment by dscho** [comment, marked as answer]
> Bash Terminal Option: MinTTY

Since you're using this option, I assume that you mean "MinTTY" by "terminal". In that case, I'm not sure and would like to redirect you to the MinTTY project over at https://github.com/mintty/mintty.

**Reply by moneyDev1111** [reply]
I see, I looked through the git installation and yes I've chosen the minTTY. and the other "native terminal" option says a lot of features will be limited. So ok I'll stick to the current one, thank you for the link, will ask it there:)
